apps/sports-helsinki/src/domain/search/eventSearch/utils.tsx

```diff
@@ -91,6 +97,19 @@ export const getTargetGroupOptions = (
     getCategoryOptions(targetGroup, TARGET_GROUP_DATA[targetGroup], t)
   );
 
+export const accessibilityProfileOptionValues = [
+  ...Object.values(AccessibilityProfile),
+] as const;
+
+export const getAccessibilityShortcomingOptions = (
+  t: TFunction,
+  accessibilities: typeof accessibilityProfileOptionValues = accessibilityProfileOptionValues
```

Comment:
Why the need for doing this differently than in getSportsCategoryOptions and getTargetGroupOptions? Not to duplicate values? BTW typeof accessibilityProfileOptionValues is readonly AccessibilityProfile[] because of the use of Object.values, if you want to have the union type of the enum's values you could make a type e.g.
```
type AccessibilityProfileValues = `${AccessibilityProfile}`
```

The shown accessibilities in the UI sketches do not include the hearing aid and the visually impaired options, for myself it is ok to leave them as they are now but I think it would be good to discuss the shown accessibilities with the designer/product owner as from earlier discussions I had I got the impression that the UI sketches in the ticket LIIKUNTA-27 show the ones wanted here.